openedx/features/caliper_tracking/utils.py

```diff
@@ -37,3 +37,30 @@ def get_username_from_user_id(user_id):
     User = get_user_model()
     user = User.objects.get(id=user_id)
     return str(user.username)
+
+
+def get_team_topic_id_team_id(team_id):
+    """
+    :param team_id: extracting from event logs
+    :return: topic_id for making team url
+    """
+    from lms.djangoapps.teams.models import CourseTeam
```

Comment:
Please run a small test. Make this app a django app.

https://docs.djangoproject.com/en/1.11/ref/applications/

And check if that fixes the issues of early imports.

Reply:
Issue not resolved

Reply:
Oka. De-prioritize this.